test/parallel/test-child-process-default-options.js

```diff
@@ -45,5 +45,5 @@ child.stdout.on('data', function(chunk) {
 
 process.on('exit', function() {
   assert.ok(response.includes('HELLO=WORLD'),
-            'spawn did not use process.env as default');
+            `spawn did not use process.env as default(process.env.HELLO = ${process.env.HELLO})`);
```

Comment:
The linter doesn't like this long line :-(

It can be fixed by shortening the line a bit or by adding an eslint directive to the top of the file (near `'use strict';`) with something like `/* eslint-disable max-len */` (you can see examples of this being done in other tests, such as `test/parallel/test-readline-keys.js` and `test/parallel/test-require-extensions-same-filename-as-dir-trailing-slash.js`.

Reply:
You could disable the rule, but you shouldn't ;-). The line needs wrapping. This can be seen locally by calling `make eslint`.

Reply:
Can you change this line to something like:

```
'spawn did not use process.env as default' +
`(process.env.HELLO = ${process.env.HELLO})`);
```

so we don't have to disable the eslint rule?